packages/core-components/src/lib/unsorted/QueryStatus.svelte

```diff
@@ -20,25 +19,37 @@
 				await invalidateAll();
 			}
 
-			statuses.push(data);
-			statuses = statuses;
+			const { status, id } = data;
+
+			const style =
```

Comment:
I think it would be best to have a predefined set of toast statuses; rather than letting component add their own. Consistency can deteriorate much more easily with this approach